lib/Listener/OutOfOfficeListener.php

```diff
@@ -53,7 +53,9 @@ public function handle(Event $event): void {
 		// check inside the sieve script acts as a redundancy.
 		$now = $this->timeFactory->getTime();
 		$enabled = $now >= $eventData->getStartDate() && $now < $eventData->getEndDate();
```

Comment:
I'm wondering if this won't produce the same issue as in the server PR for OOO responders where the date is "today". Because that would produce a timestamp that is higher that `startDate` and no event would be triggered.